workbench-core/accounts/src/services/projectService.ts

```diff
@@ -80,4 +71,204 @@ export default class ProjectService {
 
     return Promise.resolve({ data: projectsResponse.Items as unknown as Project[] });
   }
+
+  /**
+   * Creates a new project
+   *
+   * @param params - the required fields to create a new project
+   * @param user - authenticated user creating the project
+   * @returns Project object of new project
+   */
+  public async createProject(params: CreateProjectReqeust, user: AuthenticatedUser): Promise<Project> {
+    // Verify caller is an IT Admin--TODO implement after dynamic AuthZ
+    // const userGroupsForCurrentUser: string[] = await this._dynamicAuthorizationService.getUserGroups(user.id);
+    // if(userGroupsForCurrentUser.length !== 1 || userGroupsForCurrentUser[0] !== 'ITAdmin') {
+    //   throw Boom.forbidden('Only IT Admins are allowed to create new Projects.');
+    // }
+
+    // Verify project name is unique and cost center exists
+    const checkResults = await Promise.all([
```

Comment:
Nitpick: `checkResults` sounds like a verb, could we make it into a noun? 

Better, would be if we just defined the variable when the promise return.

Does this code work.
```
const [projNameInUse, costCenter] = await Promise.all([this._isProjectNameInUse(params.name),
      this._getCostCenter(params.costCenterId)
    ]);
```

Reply:
`_isProjectNameInUse` is a `void` function now and the `projNameInUse` variable is never used so ESlint complains. I will make the variable sound like a noun